frontend/src/pages/article/components/article-form/article-form.tsx

```diff
@@ -75,6 +109,14 @@ const ArticleForm: React.FC<Properties> = ({ articleForUpdate }) => {
           publishedAt: isArticlePublished ? new Date().toISOString() : null,
         };
 
+        await storage.set(StorageKey.ARTICLE_TITLE, payload.title);
+        await storage.set(StorageKey.ARTICLE_TEXT, payload.text);
+        generatedPrompt &&
+          (await storage.set(
+            StorageKey.PROMPT,
+            JSON.stringify(generatedPrompt),
+          ));
```

Comment:
This should not be on article submit - it should be on form unmount
Also should be done with actions and store (set\get all this stuff from PR)